### PR TITLE
Added image optimization survey blog post

### PR DESCRIPTION
**Pull request by samdutton**
As discussed with @kaycebasques.

### CODE REVIEW

**Comment by kaycebasques** [comment]
@samdutton I'll review this Friday at latest

File: src/site/content/en/blog/image-optimization-survey-2019/index.md [hunk 1]
```diff
@@ -0,0 +1,566 @@
...
+   direct comments from web developers, because we don't want to spread FUD or misinformation. On
+   the other hand, intermediate to advanced web developers might find the very-specific comments
+   from fellow web developers helpful. We need to investigate that further somehow. Also note how
+   I added an "Audience" section to this page. In general, all web.dev pages should clearly guide
+   web developers towards building a better site. When a page doesn't clearly do that, an "Audience"
+   section might help us make it clear why or how this page is useful to a web developer 
+   (or the web developer ecosystem). If we can't describe a clear usefulness then that's a clue that
+   the content might not be a good fit for web developers. In the future I think we need to help 
+   web developers evaluate which image optimization techniques are best for them.
+#}
```
Comment (kaycebasques) [inline src/site/content/en/blog/image-optimization-survey-2019/index.md]: @samdutton FYI

**Comment by kaycebasques** [comment]
@samdutton I didn't get to this in the timeline that I told you I would, so I just went ahead and copyedited it until it was in a publishable place. Please submit a PR on top of the merged doc if I've introduced any problems.

**Comment by samdutton** [comment]
Thanks @kaycebasques ! As you suggest, I'll take a look and do a PR if necessary.

**Pull request merged**